Package.swift

```diff
@@ -7,7 +7,9 @@ let package = Package(
     name: "VSM",
     platforms: [
         .iOS(.v13),
-        .macOS(.v11)
+        .macOS(.v11),
+        .watchOS(.v6),
+        .tvOS(.v13)
```

Comment:
are those new version aligned with iOS 13?

Reply:
yes, those are the same version that came out the same year as iOS 13